docs/classes/bpmnparser.html

```diff
@@ -240,7 +240,7 @@ <h3>
 											<li>
 												Defined in
 												<a
-													href="https://github.com/CreditSenseAU/zeebe-client-node-js/blob/46ff7a6/src/lib/BpmnParser.ts#L43"
+													href="https://github.org/CreditSenseAU/lib/BpmnParser.ts#L43"
```

Comment:
github.com

Reply:
Thanks for picking this up. Pushed this change.